maintainer/change_release.sh

```diff
@@ -73,13 +73,13 @@ echo "Using sed = $SED"
 
 echo "Setting RELEASE/__version__ in MDAnalysis to $RELEASE"
 
-git grep -E -l 'RELEASE.*[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(dev|rc*[0-9])?' $FILES  \
-   | xargs -I FILE $SED '/RELEASE/s/[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(dev|rc*[0-9])?/'${RELEASE}'/' -i.bak FILE
+git grep -E -l 'RELEASE.*[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev*[0-9]|-rc*[0-9])?' $FILES  \
+   | xargs -I FILE $SED '/RELEASE/s/[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev*[0-9]|-rc*[0-9])?/'${RELEASE}'/' -i.bak FILE
 
-git grep -E -l '__version__ =.*[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(dev|rc*[0-9])?'  $FILES \
-   | xargs -I FILE $SED '/__version__/s/[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(dev|rc*[0-9])?/'${RELEASE}'/' -i.bak FILE
+git grep -E -l '__version__=.*[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?(-dev*[0-9]|-rc*[0-9])?'  $FILES \
```

Comment:
The `*` is in the wrong place. Should be
```
(-dev[0-9]*|-rc[0-9]*)
```
so that you can have 0 to many numbers; right now you match `de0`, `devvvvvvvvvvvvvvvvv3`, etc.

(The rc one was apparently an old bug.)